ext/AvizExt/viz/spatial.jl

```diff
@@ -0,0 +1,140 @@
+import GeoMakie.GeoJSON: AbstractFeatureCollection, features, bbox
+
+# Temporary monkey-patch to support retrieval of multiple features
+Base.getindex(fc::AbstractFeatureCollection, i::UnitRange) = features(fc)[i]
+Base.getindex(fc::AbstractFeatureCollection, i::Vector) = features(fc)[i]
+
+
+"""
+    create_map!(f, geodata, data, highlight, centroids, c_label)
+
+Create a spatial choropleth figure.
+
+# Arguments
+- `f` : GLMakie figure to create plot in
+- `geodata` : FeatureCollection, Geospatial data to display
+- `data` : Vector, values to use for choropleth
+- `highlight` : Vector, stroke colors for each location
+- `centroids` : Vector{Tuple}, of lon and lats
+- `c_label` : String, label to use for color bar
+"""
+function create_map!(f, geodata, data, highlight, centroids, c_label)
+    lon = first.(centroids)
+    lat = last.(centroids)
+
+    map_buffer = 0.025
+    spatial = GeoAxis(
+        f[1, 1];
+        lonlims=(minimum(lon) - map_buffer, maximum(lon) + map_buffer),
+        latlims=(minimum(lat) - map_buffer, maximum(lat) + map_buffer),
+        title="Study Area",
```

Comment:
Might be good to allow optional title name input?

Reply:
Probably